### PR TITLE
Add Ledger support

### PR DESCRIPTION
**Pull request by webmaster128**
this probably works in Chromium-based browsers only due to WebUSB support.

`yarn start-https` must be used for local testing of WebUSB

### CODE REVIEW

**Comment by webmaster128** [comment]
Preview here:

Bildschirmfoto 2019-08-28 um 18 15 08
Bildschirmfoto 2019-08-28 um 18 15 18
Bildschirmfoto 2019-08-28 um 18 17 09

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "lint": "eslint -c .eslintrc.json --max-warnings 0 'src/**/*.ts{,x}'",
     "lint-fix": "yarn lint --fix",
     "start": "react-scripts start",
+    "start-https": "HTTPS=true react-scripts start",
```
Comment (willclarktech) [inline package.json]: Why not make this the default `start` script?

Comment (webmaster128) [reply package.json]: Clicking through all the certificate warnings can be very annoying for a user but also during rapid development. I think the easy part should be easy and if someone really needs ledger support locally, this is how they get it.

File: custom_types/ledgerhq__hw-transport/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+declare module "@ledgerhq/hw-transport" {
+  // See https://github.com/LedgerHQ/ledgerjs/blob/master/packages/hw-transport/src/Transport.js
```
[the remark this answers is not in the document]
Comment (webmaster128) [reply custom_types/ledgerhq__hw-transport/index.d.ts]: If we are lucky, we can remove this file very very soon (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/38001) 🤞

**Pull request merged**